lib/railway_ipc/consumed_message.ex

```diff
@@ -1,30 +1,17 @@
-defmodule RailwayIpc.Core.MessageAccess do
+defmodule RailwayIpc.ConsumedMessage do
```

Comment:
`MessageAccess` functions are moved into the `ConsumedMessage` and `PublishedMessage` contexts respectively. Entrypoint of app, `RailwayIpc` module, calls out to these contexts.